app/src/component/storageStatsCard/view.js

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import { normalizeURI, parseURI } from 'lbry-redux';
+import {
+  ActivityIndicator,
+  Platform,
+  Switch,
+  Text,
+  TouchableOpacity,
+  View
+} from 'react-native';
+import { formatBytes } from '../../utils/helper';
```

Comment:
We really need to get aliases or resolves added to remove these.
I die a little inside every time I see these crazy paths, assign an issue to me if you need me to work on this.

Reply:
I plan to sort this out later. I just haven't taken the time to figure out how to do this yet.